### ISSUE TITLE
new backbone

### ISSUE THREAD

**Issue by parul12111** [zhanght021/RPT#13]
Hi! Thank you for this amazing repository. I want to build my own backbone and integrate it into rpt, what are the necessary steps and can you please list all the files and directories where I will need to make changes. Thank you!

**Comment by zhanght021** [comment]
Hello，  I prepare to sort out the whole code and open source , but recently my company is busy, so there is  not plenty of time to make training code open source, so there is no training code yet.
But the training code is easy, it is modified from reppoints and siamrpn++
So the first thing you need to do is writing the training code.

If you want to change backbone , you should add the backbone file into reppoints->models->backbone, modify the __init__.py in reppoints->models->backbone, and also you should modify the config.yaml,  according to the output size of feature map, yor should modify '__C.TRAIN.OUTPUT_SIZE' in core->config.py

**Comment by parul12111** [comment]
ok thank you

**Comment by parul12111** [comment]
I am trying to changed the backbone from resnet50 to resnet101.
but when I run eval.py I got this error-
python eval1.py    --tracker_path ./results    --dataset VOT2018   --tracker_prefix siamreppoints
loading VOT2018: 100%|██████████████████████████████████| 60/60 [00:00<00:00, 78.59it/s, zebrafish1]
eval ar:   0%|                                                                | 0/1 [00:00<?, ?it/s]C:\Users\jinda\OneDrive\Desktop\RPT-master\toolkit\utils\statistics.py:54: RuntimeWarning: Mean of empty slice
  acc = np.nanmean(overlaps)
C:\Users\jinda\OneDrive\Desktop\RPT-master\toolkit\utils\statistics.py:54: RuntimeWarning: Mean of empty slice
  acc = np.nanmean(overlaps)

**Comment by parul12111** [comment]
it is not evaluating accuracy and eao
eval ar: 100%|████████████████████████████████████████████████████████| 1/1 [00:00<00:00,  2.97it/s]
eval eao:   0%|                                                               | 0/1 [00:00<?, ?it/s]C:\Users\jinda\OneDrive\Desktop\RPT-master\toolkit\utils\statistics.py:54: RuntimeWarning: Mean of empty slice
  acc = np.nanmean(overlaps)
eval eao: 100%|███████████████████████████████████████████████████████| 1/1 [00:12<00:00, 12.00s/it]
---------------------------------------------------------------
| Tracker Name  | Accuracy | Robustness | Lost Number |  EAO  |
---------------------------------------------------------------
C:\Users\jinda\OneDrive\Desktop\RPT-master\toolkit\evaluation\ar_benchmark.py:73: RuntimeWarning: Mean of empty slice
  accuracy = np.nanmean(overlaps)
| siamreppoints |   nan    |   16.735   |   3574.0    | 0.000 |
---------------------------------------------------------------

**Comment by zhanght021** [comment]
Hello, the backbone resnet50 we used is from siamrpn++, they remove the padding in the 'self.conv1 = nn.Conv2d(3, 64, kernel_size=7, stride=2, padding=0,  bias=False)',  also they change the stride from 16, 32 of resnet50's layer4 and layer5 to 8, 8
